pypsa/linopf.py

```diff
@@ -1038,5 +1040,12 @@ def save_optimal_capacities(n, iteration, status):
     n.links[['p_nom', 'p_nom_extendable']] = n.links['p_nom_opt'], False
     kwargs['warmstart'] = False
     network_lopf(n, snapshots, **kwargs)
+    if track_iterations:
+        n.global_constraints = n.global_constraints.rename(columns={'mu':f'mu_{iteration}'})
+    if 'lv_limit' in n.global_constraints.index:
+        logger.info(f'Resulting dual value for line volume limit (lv_limit) set to the one of iteration {iteration-1} '
+                    'due to its optimisation process.') # otherwise, final dual for lv_limit is NaN.
+        if track_iterations: n.global_constraints.at['lv_limit', f'mu_{iteration}'] = final_duals['lv_limit']
```

Comment:
```suggestion
        if track_iterations:
            n.global_constraints.at['lv_limit', f'mu_{iteration}'] = final_duals['lv_limit']
```
style comment: would indent here

I was wondering whether it would make sense to keep the `mu_{iteration}` of the last iteration as NaN and not override it (so it becomes clear that you use the "last iteration" heuristic).

Reply:
If the iterations are being tracked, I was thinking that as well. However, if not, I'd overwrite it.